lib/onceover/controlrepo.rb

```diff
@@ -346,9 +346,18 @@ def fixtures
 
     def hiera_config_file
       # try to find the hiera.yaml file
-      hiera_config_file = File.expand_path('./hiera.yaml', @spec_dir) if File.exist?(File.expand_path('./hiera.yaml', @spec_dir))
-      hiera_config_file = File.expand_path('./hiera.yaml', @root)     if File.exist?(File.expand_path('./hiera.yaml', @root))
-      hiera_config_file
+      possible_locations = [
+        './spec/hiera.yaml',
+        './hiera.yaml'
+      ]
+
+      # Keep all the ones that exist
+      possible_locations.keep_if do |location|
+        File.exist?(File.expand_path(location, @root))
+      end
+
+      # Return the location, or nil
+      possible_locations.first ? File.expand_path(possible_locations.first) : nil
```

Comment:
@jessereynolds What about this?

Reply:
Well with that approach you're still testing the existence of both paths unnecessarily, hence I think the case statement approach is better (and a lot less code). You don't like the case statement approach?

Reply:
Oh yeah right, like a case statement and a return, hold on.